x-pack/plugins/osquery/public/results/translations.ts

```diff
@@ -7,6 +7,8 @@
 
 import { i18n } from '@kbn/i18n';
 
+export const OSQUERY_APP_NAME = 'Osquery';
```

Comment:
can we use https://github.com/patrykkopycinski/kibana/blob/main/x-pack/plugins/osquery/common/index.ts#L11 ?